packages/aws-cdk-migration/lib/rewrite.ts

```diff
@@ -108,6 +113,12 @@ function updatedLocationOf(modulePath: string, options: RewriteOptions): string
     return customModulePath;
   }
 
+  if (options.isInAlphaPackage && modulePath.endsWith('generated')) {
+    const modulePathSplit = modulePath.split(/[.\/]/);
```

Comment:
The forward-slash doesn't need to be escaped here.

```suggestion
    const modulePathSplit = modulePath.split(/[./]/);
```